schema_utils_test.py

```diff
@@ -559,6 +575,9 @@ def test_schemas_are_correctly_validated(self):
             'ui_config': {
                 'coding_mode': 'python',
             }
+        }, {
+            'type': 'object_dict',
+            'validate_method': 'arbitary_method'
```

Comment:
If the method does not exists then it should raise an error.

Reply:
In this case it is just checking the schema, i.e., the value of the validate_method key should be string, it is not validating any object_dict.
Please correct me if I am getting wrong.

Reply:
This is just checking the schema i.e., the value to validate_method key must be string, it is not valildating any object_dict.
Please correct me if I am wrong.

Reply:
Yep, I got it. Thanks.